frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -141,6 +142,16 @@ export default function SideNavigation({
               title='API'
               icon={<LinkIcon />}
             />
+            <NavMenuItem
+              href='/docs/python/index.html'
+              selectedPage={page}
+              primaryText='Python Client Docs'
+              drawerOpen={drawerOpen}
+              menuPage='pythonDocs'
+              title='Python Client Docs'
+              icon={<DescriptionIcon />}
+            />
+            <Divider />
```

Comment:
This divider should be moved back to where it was before, below the help and support `NavMenuItem`.